coq-corn.opam

```diff
@@ -40,10 +40,7 @@ CoRN includes the following parts:
   integrals, graphs of functions, differential equations.
 """
 
-build: [
-  ["./configure.sh"]
-  [make "-j%{jobs}%"]
-]
+build: [make "-j%{jobs}%" ]
```

Comment:
This is because of this change. Unfortunately, the coq-community templates do not support yet the addition of this `./configure.sh` step, so you have to restore it and only maintain the line change regarding the license identifier.

Reply:
Done